packages/vtree/core/modeling/primitives/rectangle.js

```diff
@@ -0,0 +1,49 @@
+
+// TODO: FORMAL DEFINITION VS IMPLEMENTATION
+// GEOM: implementation
+// anything before that: formal
+
+/**
+ * Construct an axis-aligned rectangle with four sides and four 90-degree angles.
+ * @param {Object} [options] - options for construction
+ * @param {Array} [options.center=[0,0]] - center of rectangle
+ * @param {Array} [options.size=[2,2]] - dimension of rectangle, width and length
+ * @returns {geom2} new 2D geometry
+ *
+ * @example
+ * let myshape = rectangle({center: [5, 5, 5], size: [10, 20]})
+ */
+const rectangle = (options) => {
```

Comment:
What!?!

This code is almost more that modeling.

Reply:
done